fhirpipe/cli/run.py

```diff
@@ -29,34 +30,30 @@
 from fhirpipe.load.references import build_identifier_dict, bind_references
 
 
-def run():
+def run(
+    connection,
+    mapping,
+    source,
+    resources,
+    reset_store,
+    chunksize,
+    bypass_validation,
+    multiprocessing,
+):
     """
     """
-    # Parse arguments
-    args = parse_args()
-
-    print(WELCOME_MSG)
-
     # Launch timer
     start_time = time.time()
 
-    # Define global config
-    set_global_config(config_path=args.config)
-
-    # Setup logging configuration
-    setup_logging()
-
     # Get the resources we want to process from the pyrog mapping for a given source
-    resources = get_mapping(
-        from_file=args.mapping, source_name=args.source, selected_resources=args.resources,
-    )
+    resources = get_mapping(from_file=mapping, source_name=source, selected_resources=resources)
 
     fhirstore = get_fhirstore()
-    if args.reset_store:
+    if reset_store:
```

Comment:
I think it's ok to keep this here for now because it's super useful in dev, but maybe we'll put this somewhere else (like in an other route) later because it sounds dangerous to be able to erase all data from the DB with a single HTTP call.